content/rdi/architecture.md

```diff
@@ -9,59 +9,89 @@ headerRange: "[2]"
 aliases: 
 ---
 
-Redis Data Integration is a product for data ingestion from existing systems of record (mainly databases) into Redis Enterprise.
-You can think of Redis Data Integration as a streaming ELT process:
+Redis Data Integration (RDI) is a product that helps Redis Enterprise users ingest and export data in near real time. RDI provides the following:
 
-- Data is **E**xtracted from the source database using [Debezium](https://debezium.io/) connectors.
+- End to end solution - No need for additional tools and integrations
+- Capture Data Change (CDC) included
+- Covers most popular databases as sources and targets
+- Declarative data mapping and transformations - no custom code needed
+- Data delivery guaranteed (at least once)
+- Zero downtime day 2 operations (reconfigure & upgrade)
+
+Redis Data Integration currently supports two use cases:
+
+- Ingest (cache prefetch).
+- Write Behind.
+
+## Ingest functionality and architecture
+
+You can think of RDI Ingest as a streaming ELT process:
+
+- Data is **E**xtracted from the source database using [Debezium Server](https://debezium.io/) connectors.
 - Data is then **L**oaded into Redis Data Integration, a Redis database instance that keeps the data in [Redis streams](https://redis.io/docs/manual/data-types/streams/) alongside required metadata.
 - Data is then **T**ransformed using provided [RedisGears recipes](https://developer.redis.com/howtos/redisgears/) and written to the target Redis DB.
 
-Redis Data Integration using Debezium works in 2 modes:
+Redis Data Integration using Debezium Server works in two modes:
 
 1. Initial sync - Where a snapshot of the entire database or a subset of selected tables is used as a baseline. The entire dataset is streamed to Redis Data Integration and then transformed and written into the target Redis database.
 2. Live updates - Where Debezium captures changes to the data that happen after the baseline snapshot and streams them to Redis Data Integration where they are transformed and written to the target.
 
 ![High-level Redis Data Integration architecture](/images/rdi/redis-di-simplified.png)
 
-## Supported data transformations
+### Supported data transformations
+
+#### Mode mapping
+
+RDI supports conversion of a database record into the following Redis types:
+
+- Hash
+- Set
+- Sorted Set
+- Stream
+- JSON
+- String
```

Comment:
I'd make each one of them a link to the proper Redis documentation page.

Reply:
done